shared/uavobjectdefinition/hwnaze.xml

```diff
@@ -11,7 +11,7 @@
 
 		<field name="GyroRange" units="deg/s" type="enum" elements="1" options="250,500,1000,2000" defaultvalue="1000"/>
 		<field name="AccelRange" units="*gravity m/s^2" type="enum" elements="1" options="2G,4G,8G,16G" defaultvalue="8G"/>
-		<field name="MPU6050Rate" units="" type="enum" elements="1" options="200,333,500" defaultvalue="500"/>
+		<field name="MPU6050Rate" units="" type="enum" elements="1" options="200,333,500" defaultvalue="333"/>
```

Comment:
The board barely does-- cpu warning.  And you definitely can't autotune @ 500Hz.